### PR TITLE
Added OMS Solution Snippit

### PR DESCRIPTION
**Pull request by bentaylorwork**

### CODE REVIEW

File: VSCode/armsnippets.json [hunk 1]
```diff
@@ -1081,4 +1081,35 @@
...
     ],
     "description": "Windows Virtual Machine"
-}
+},
+"OMS Solution": {
+    "prefix": "arm-oms-solution",
+    "body": [
+        "{",
+        "   \"name\": \"[concat(${solutionName}, [${workSpaceName}])]\",",
+        "   \"location\": \"${West Europe}\",",
```
Comment (philon-msft) [inline VSCode/armsnippets.json]: Please use [resourceGroup().location] for the location

File: VSCode/armsnippets.json [hunk 2]
```diff
@@ -1081,4 +1081,35 @@
...
+        "   \"tags\": {  },",
+        "   \"type\": \"Microsoft.OperationsManagement/solutions\",",
+        "   \"apiVersion\": \"2015-11-01-preview\",",
```
Comment (philon-msft) [inline VSCode/armsnippets.json]: Is there a newer apiVersion you can use?

Comment (bentaylorwork) [inline VSCode/armsnippets.json]: It's the latest one, see attached.

image

**Comment by philon-msft** [comment]
Thanks, @bentaylorwork!

**Pull request merged**